s4e-web/src/app/components/login/login.component.ts

```diff
@@ -9,7 +9,7 @@ import {untilDestroyed} from 'ngx-take-until-destroy';
 import {disableEnableForm} from '../../utils/miscellaneous/miscellaneous';
 
 @Component({
-  selector: 's4e-login',
+  selector: '[s4e-login]',
```

Comment:
Better leave the tag as the selector. The other components do it that way and I don't see a reason to do it differently here.
Also: [Angular style guide](https://angular.io/guide/styleguide#style-05-03)